src/content/tutorials/connectivity/coredns/index.md

```diff
@@ -4,65 +4,42 @@ title: Advanced CoreDNS configuration
+  principal:
+    parent: tutorials-connectivity
+    identifier: tutorials-connectivity-coredns
+aliases:
+  - /advanced/connectivity/coredns
+  - /guides/advanced-coredns-configuration/
+  - /advanced/coredns/
 user_questions:
   - How can I override the default CoreDNS configuration?
   - How can I customize the CoreDNS configuration?
   - How can I adjust resource limits for CoreDNS?
   - Where is the user values ConfigMap for CoreDNS?
-last_review_date: 2023-11-07
-aliases:
-  - /advanced/connectivity/coredns
-  - /guides/advanced-coredns-configuration/
-  - /advanced/coredns/
+last_review_date: 2025-08-04
 owner:
   - https://github.com/orgs/giantswarm/teams/team-phoenix
 ---
 
-Your Giant Swarm installation comes with a default configuration for the [CoreDNS addon](https://github.com/coredns/coredns)
-
-You can override these defaults in a ConfigMap named `coredns-user-values` in the management cluster.
-
-__Note__: In Cluster API, CoreDNS is part of default apps `<provider>` chart (example [Cluster API for Openstack](https://github.com/giantswarm/default-apps-openstack/). In order to extend the app, you need to patch the App Custom Resource to use `coredns-user-values` configmap. [Check out this example](https://github.com/giantswarm/gitops-template/pull/74/files).
+Your Giant Swarm clusters come with a default configuration for the [CoreDNS add-on](https://github.com/coredns/coredns). CoreDNS is installed as default application in your clusters using a [HelmRelease](https://fluxcd.io/flux/components/helm/helmreleases/). This guide explains how you can customize the CoreDNS configuration in your clusters.
 
-## Where is the user values ConfigMap
-
-Given the cluster you are trying to configure has id: `123ab` then you will find the `coredns-user-values` ConfigMap in the management cluster in the `123ab` namespace:
-
-```nohighlight
-$ kubectl -n 123ab get cm coredns-user-values --context=control-plane
-NAME                                   DATA      AGE
-coredns-user-values                    0         11m
-```
+## Where to store the user configuration
 
-__Warning:__
-
-Please do not edit any other CoreDNS related ConfigMaps.
-
-Only the user values ConfigMap is safe to edit.
-
------
-
-## How to set configuration options using the user values ConfigMap
-
-On the management cluster, create or edit a ConfigMap named `coredns-user-values`
-in the workload cluster namespace as described above:
+Given the cluster you are trying to configure is called: `123ab` you can create or extend the `<CLUSTER>-user-values` ConfigMap of the organization namespace on the management cluster. Inside the ConfigMap, the Helm values are passed as the field called `values` which lets you define a
 
 ```yaml
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  labels:
-    app: coredns
-  name: coredns-user-values
-  namespace: abc12
 data:
   values: |
-    configmap:
-      cache: "60"
+    global:
+      ...
+kind: ConfigMap
+metadata:
+  name: 123ab-user-values
```

Comment:
there's no configmap called this way. kubectl-gs uses the format `<cluster_id>-userconfig`. Although there are customers using a different structure, I'd say we stick to our default formats.